virttest/bootstrap.py

```diff
@@ -360,6 +365,20 @@ def _forced_or_detected(forced, detected):
         cfg.write("                            - @%s:\n" % release)
         cfg.write("                                variants:\n")
         cfg.write("                                    - @%s:\n" % arch)
+        cfg.write("variants:\n")
+        cfg.write("    - @Hostvendor:\n")
+        cfg.write("        variants:\n")
+        cfg.write("            - @%s:\n" % vendor)
+        if family:
+            cfg.write("variants:\n")
```

Comment:
Could you please elaborate on why it is not being a sub-variant of cpu vendor?

Reply:
just wanted to use them independently, 

only Hostvendor.intel
only Hostvendor.amd
are one possibilities

only Hostfamily.power9
only Hostfamily.powe8.2.1 
can be another possibility, it helps user to choose between different filters with name properly

Reply:
Sounds reasonable, btw, we can also implement it with using [named-variants](https://avocado-vt.readthedocs.io/en/latest/CartesianConfig.html#named-variants).